packages/ember-engines/app/engine.js

```diff
@@ -0,0 +1 @@
+export { default } from 'ember-engines/engine';
```

Comment:
do we need to do an app export here? I assume we can rely on addon exports properly if it's intended to be imported explicitly?